tools/jenkins/slaves/Dockerfile.centos

```diff
@@ -40,9 +38,10 @@ RUN scl enable rh-ror50 $HOME/bushslicer/tools/hack_bundle.rb
 #   security concern because of exposing VPN details
 # RUN sed -i -e 's/^\s\+curl\s/  curl -Sk /' /usr/local/bin/run-jnlp-client
 
+# Cleanup and remove any instance of the private repository
```

Comment:
I htink we can remove this line, a little confusing on top of many things while it describes only one thing. Command is clear by itself IMO.